include/reporters/catch_reporter_console.hpp

```diff
@@ -258,11 +258,14 @@ namespace Catch {
             }
         }
         void lazyPrintRunInfo() {
+            // Usefulness of this message is questionable.
```

Comment:
Thanks for noticing this! I forget disabling it (it's annoying to see whole message over and over again if you do `make check` several times in a row).